lib/browser/init.ts

```diff
@@ -152,7 +152,9 @@ app._setDefaultAppPaths(packagePath)
 require('@electron/internal/browser/devtools')
 
 // Load the chrome extension support.
-require('@electron/internal/browser/chrome-extension')
+if (!process.electronBinding('features').isExtensionsEnabled()) {
```

Comment:
We should add similar checks around require calls to `chrome-api` in the renderer (the thing that binds `global.chrome`)